app/helpers/application_helper.rb

```diff
@@ -1648,12 +1648,8 @@ def listicon_image_tag(db, row)
       vendor = host ? host.vmm_vendor_display.downcase : "unknown"
       image = "svg/vendor-#{vendor}.svg"
     elsif db == "MiqAction"
-      action = @targets_hash[@id.to_i]
-      image = if action && action.action_type != "default"
-                "100/miq_action_#{action.action_type}.png"
-              else
-                "100/miq_action.png"
-              end
+      action = @targets_hash[row['id']]
```

Comment:
This is a fonticon-decorator PR, others will be fixed in separate ones if that's ok :wink: